### DISCUSSION TITLE
Stable Edge on Android now supports Better Xcloud

### DISCUSSION THREAD

**Discussion by RC1992Jules** [redphx/better-xcloud#555] [General]
After the recent changes to the Xbox app and Xbox game pass app I decided to do some looking into on whether or not extensions were expanded upon in the stable and standard version of Microsoft Edge on Android. I noticed that tampermonkey was able to be installed now as an extension on Microsoft Edge Android. After installing that, I was able to install the better XCloud script just like I would on a standard Windows PC using an edge browser and all of the features work perfectly from what I can tell. I am curious though on how this could possibly change the performance or development approaches to having better Xcloud run on Android devices using the browser. Is this way of using Better XCloud Superior to the kiwi version or even the native app that's currently being developed? Is there even a need for a separate app on Android now or should I just use the browser version since it'll be on par with the latest browser patches and updates? Sorry, I'm not a developer, just very curious because this really interests me.

**Comment by KingNothing81** [comment]
Now that stable Edge supports extensions on Android, it should definitely be the better choice for most users still using Kiwi Browser.

But may I ask why you don’t use or prefer using the BxC Android app? Do you have the feeling that Edge performs better? 

As mentioned on the issue tracker, I wonder if Microsoft will do some xCloud specific modifications to mobile Edge or might also make “Clarity Boost” available on phones and tablets now. In that case, using it might be superior to using the app.

**Reply by RC1992Jules** [reply]
Oh I actually really enjoy the BxC app and use it frequently. 
I just really enjoy seeing all of the changes and development stages of this project so I like to browse around and see what changes are being made to all aspects. 
There is a very specific occurrence that will cause me to choose the standard Xbox apps over the BxC app and I'm not sure how to describe this so I'll try my best to. 
When using the standard Xbox apps to stream games I noticed that the streams will sacrifice image quality in order to keep the feel of the game running smoothly. For certain games, I enjoy this a lot better then what I experience with the BxC app because when I use the BxC app The stream will stutter and controls will be lost for brief moments, but the image stays sharp. 
So for me what I'm playing a slower paced game I find that BxC is better for me, but when I'm playing a faster paced game like a shooter I enjoy the experience better when the gameplay stays more consistent.

And I'm currently wondering that too since you brought it up and it makes me want to test running the standard Xbox.com/play web page and desktop mode to see if any desktop features pop up. I doubt it, but I still might try because I'm curious.

**Reply by KingNothing81** [reply]
The BxC app uses the Android System Webview by default. And although it’s not the same, Edge and the System Webview are both based on Chromium. 

The Android System Webview also gets updates quite frequently directly over the PlayStore. Independently from the BxC app itself. So if xCloud works well with the default Webview, I don’t see a real advantage in using Edge. Especially for Android TV users (like me) that would need to sideload Edge and need mouse support to set things up.

But please share your experience with Edge. But I think just switching to desktop mode won’t make a difference. But maybe changing user agent profile to “Windows + Edge” in the BxC settings.

**Comment by redphx** [comment]
The guide has been updated: https://better-xcloud.github.io/android-browser/

The app has more features. Only use browser method if you have problem with the app.
